README.md

```diff
@@ -1,16 +1,17 @@
-[![PyPi](https://badge.fury.io/py/matplotlib.svg)](https://badge.fury.io/py/matplotlib)
-[![Downloads](https://pepy.tech/badge/matplotlib/month)](https://pepy.tech/project/matplotlib)
+[![PyPi](https://img.shields.io/pypi/v/matplotlib)](https://pypi.org/project/matplotlib/)
+[![Conda](https://img.shields.io/conda/vn/conda-forge/matplotlib)](https://anaconda.org/conda-forge/matplotlib)
+[![Downloads](https://img.shields.io/pypi/dm/matplotlib)](https://pypi.org/project/matplotlib)
 [![NUMFocus](https://img.shields.io/badge/powered%20by-NumFOCUS-orange.svg?style=flat&colorA=E1523D&colorB=007D8A)](https://numfocus.org)
 
 [![DiscourseBadge](https://img.shields.io/badge/help_forum-discourse-blue.svg)](https://discourse.matplotlib.org)
 [![Gitter](https://badges.gitter.im/matplotlib/matplotlib.svg)](https://gitter.im/matplotlib/matplotlib)
 [![GitHubIssues](https://img.shields.io/badge/issue_tracking-github-blue.svg)](https://github.com/matplotlib/matplotlib/issues)
-[![GitTutorial](https://img.shields.io/badge/PR-Welcome-%23FF8300.svg?)](https://git-scm.com/book/en/v2/GitHub-Contributing-to-a-Project)
+[![GitTutorial](https://img.shields.io/badge/PR-Welcome-%23FF8300.svg?)](https://matplotlib.org/stable/devel/index.html)
```

Comment:
This doesn't seem to be a Git tutorial.

Reply:
Yes, you are correct. Where does this text actually show up? I'm not saying that it shouldn't be correct though.

Should we also add a Git Tutorial bagde? Like https://img.shields.io/badge/git-tutorial-brightgreen

Reply:
No. IMHO that's too specific. Git explanations are/should be part of the contribution guide. This link to the contribution guide is enough here.

Reply:
> Yes, you are correct. Where does this text actually show up? I'm not saying that it shouldn't be correct though.

It appears in the image's alt text.

Reply:
That is what I thought, but as part of that I have now realized that my browser doesn't show alt-texts at all...